Validation/RecoMuon/plugins/ME0MuonTrackCollProducer.h

```diff
@@ -0,0 +1,35 @@
+
+#ifndef RecoMuon_ME0MuonTrackCollProducer_h
+#define RecoMuon_ME0MuonTrackCollProducer_h
+
+#include <memory>
+#include "FWCore/Framework/interface/EDProducer.h"
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "DataFormats/MuonReco/interface/MuonFwd.h"
+#include "DataFormats/MuonReco/interface/ME0Muon.h"
+#include "DataFormats/MuonReco/interface/ME0MuonCollection.h"
+#include "DataFormats/CSCRecHit/interface/CSCSegmentCollection.h"
+#include "DataFormats/DTRecHit/interface/DTRecSegment4DCollection.h"
+#include "DataFormats/HepMCCandidate/interface/GenParticle.h"
+
+class ME0MuonTrackCollProducer : public edm::EDProducer {
```

Comment:
all new modules should be using thread - aware base classes.
edm::stream::EDProducer appears to be appropriate